Makefile.PL

```diff
@@ -32,7 +32,7 @@ our $TEXMF;
 my ($texmfspec) = grep { /^TEXMF/ } @ARGV;
 if ($texmfspec && $texmfspec =~ /^TEXMF\s*=(.*)$/) {
   $TEXMF = $1;
-  @ARGV = grep { $_ ne $texmfspec } @ARGV;    # Remove so MakeMaker doesn't fret.
+  local @ARGV = grep { $_ ne $texmfspec } @ARGV;    # Remove so MakeMaker doesn't fret.
```

Comment:
doesn't local here restrict the effect to the body of the if? If so, it's pretty much a NOOP (& doesn't have desired effect)

Reply:
Indeed, this was a harmful change, apologies. It bugs me that removing it once again makes any change to Makefile.PL requires `--no-verify` on commit, but I shouldn't have broken it. Critic's insisting messages pushed me too far.